graphql/src/main/java/com/linecorp/armeria/server/graphql/DefaultGraphqlService.java

```diff
@@ -53,17 +50,20 @@ final class DefaultGraphqlService extends AbstractGraphqlService implements Grap
 
     private final boolean useBlockingTaskExecutor;
 
+    private final GraphqlErrorHandler errorsHandler;
+
     DefaultGraphqlService(GraphQL graphQL, DataLoaderRegistry dataLoaderRegistry,
-                          boolean useBlockingTaskExecutor) {
+                          boolean useBlockingTaskExecutor, GraphqlErrorHandler errorsHandler) {
         this.graphQL = requireNonNull(graphQL, "graphQL");
         this.dataLoaderRegistry = requireNonNull(dataLoaderRegistry, "dataLoaderRegistry");
         this.useBlockingTaskExecutor = useBlockingTaskExecutor;
+        this.errorsHandler = errorsHandler;
     }
 
     @Override
     protected HttpResponse executeGraphql(ServiceRequestContext ctx, GraphqlRequest req) throws Exception {
```

Comment:
I think the `AbstractGraphqlService` should also be changed. WDYT? 🤔 

ex) https://github.com/line/armeria/blob/master/graphql-protocol/src/main/java/com/linecorp/armeria/server/graphql/protocol/AbstractGraphqlService.java#L71